contracts/staking/README.md

```diff
@@ -34,9 +34,11 @@ Validator exit is initiated with a call to `initializeEndValidation` on the Vali
 
 The `SetSubnetValidatorWeightMessage` is delivered to the P-Chain as the payload of a [`SetSubnetValidatorWeightTx`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#setsubnetvalidatorweighttx). The P-Chain acknowledges validator exit by signing either a `SetSubnetValidatorWeightMessage` with the `weight=0`, or a `SubnetValidatorRegistrationMessage` with `valid=0`. `completeEndValidation` is then called on the Validator Manager contract, with `setWeightMessageType` set according to the Warp message type being delivered. The validation is removed from the contract's state, and for PoS Validator Managers, staking rewards are disbursed and stake is returned.
 
-#### Exit the Validator Set Directly on the P-Chain
+#### Disable a Validator Directly on the P-Chain
 
-ACP-77 also provides a method for validators to exit a Subnet's validator set without interacting with the Subnet directly. The P-Chain transaction [`ExitValidatorSetTx`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#exitvalidatorsettx) removes the validator from the Subnet validator set tracked on the P-Chain. The P-Chain will sign a `SetSubnetValidatorWeightMessage` or a `SubnetValidatorRegistrationMessage`, which may then be provided to the Validator Manager contract via a call to `completeEndValidation`, as described above. Note however that without the uptime proof provided in the initial call to `initializeEndValidation`, the Validator Manager will not credit any staking rewards to the validator.
+ACP-77 also provides a method to disable a validator without interacting with the Subnet directly. The P-Chain transaction [`DisableValidatorTx`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#disablevalidatortx) disables the validator on the P-Chain. The disabled validator's weight will still count towards the Subnet's total weight. 
+
+Disabled Subnet Validators can re-activate at any time by increasing their balance with an `IncreaseBalanceTx`. Anyone can call `IncreaseBalanceTx` for any validator on the P-Chain. A disabled validator can only be totally removed from the validator set by a call to `initializeEndValidation`.
```

Comment:
This actually isn't quite correct. 

I a validator's balance runs out on the P-Chain, then they are marked as inactive and anyone can increase their balance to top it up. However after a `DisableValidatorTx`, the only change to that validator that is allow is `SetValidatorWeight = 0`, to completely remove it. You cannot reactive a validator after `DisabelValidatorTx` though.

Reply:
I think in the update to ACP77 `DisableValidatorTx` is the same as any other inactive valiator and can be re activated with `IncreaseBalaanceTx`. https://github.com/avalanche-foundation/ACPs/pull/144/files#diff-d2befccafcc84a6c9bf56f76def8e04647ed5087fe8480f28de11fea63fb769aR268

Reply:
Did not realize this! 

Thank you correcting me here